src/shared/nt-results/ResultsPage.react.js

```diff
@@ -19,6 +19,16 @@ class ResultsPage extends React.Component {
       _id: PropTypes.string.isRequired,
       url: PropTypes.string.isRequired,
     })).isRequired,
+    features: PropTypes.arrayOf(PropTypes.shape({
+      _id: PropTypes.string.isRequired,
+      name: PropTypes.string.isRequired,
+    })).isRequired,
+  }
+
+  state = { selectedFeatureId: undefined }
+
+  handleSelectFeature = featureId => () => {
+    this.setState({ selectedFeatureId: featureId })
   }
```

Comment:
The `featureId` receive from params is not **selected** yet. It is **going to be selected**.